packages/expo-keep-awake/src/KeepAwake.types.ts

```diff
@@ -11,3 +11,14 @@ export enum KeepAwakeEventState {
 
 // @needsAudit
 export type KeepAwakeListener = (event: KeepAwakeEvent) => void;
+
+export type KeepAwakeOptions = {
+  /**
+   * The call will throw an unhandled promise rejection on Android when the original Activity is dead or deactivated.
+   * Set the value to `true` for suppressing the uncaught exception.
+   */
+  suppressDeactivateWarnings?: boolean;
+
+  /** A callback that is invoked when the keep-awake state changes (web-only). */
```

Comment:
```suggestion
  /** 
   * A callback that is invoked when the keep-awake state changes.
   * @platform web
   */
```

We can use the `@platform` tag here.